### PR TITLE
circle progress bar component

### PR DESCRIPTION
**Pull request by nabil386**
### Description
circle progress bar component
List of proposed changes:

- CircleProgressBar.js
-

### CODE REVIEW

**Review by amirpourpanahi — APPROVED** [review]

File: styles/stepper.css [hunk 1]
```diff
@@ -104,3 +104,36 @@
         padding-left: 1rem;
     }
 }
+
+.circle-progress-indicator {
```
Comment (arsanig) [inline styles/stepper.css:108]: This shouldn't be a part of the `stepper.css` file and would be more maintainable if it's in its own file.

File: styles/stepper.css [hunk 2]
```diff
@@ -104,3 +104,36 @@
...
+    font-family: Lato;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 30px;
+    color: #259F46;
+    letter-spacing: 0.1px;
+}
+
+.RCP {
```
Comment (arsanig) [inline styles/stepper.css:130]: What's RCP? Can this `classname` be made more descriptive so it's easier to identify what it's targeting in the future?

File: styles/stepper.css [hunk 3]
```diff
@@ -104,3 +104,36 @@
...
+    width: 50px;
+    margin-left: 0px;
+
+}
+
+svg {
```
Comment (arsanig) [inline styles/stepper.css:136]: Targeting all `<svg>` attributes is not a good practice. Tailwind prefers an inline svgs with a `class=` attribute to modify svgs

**Review by arsanig — CHANGES_REQUESTED** [review]
Please see feedback below

File: styles/stepper.css (hunk 1 above)
Comment (nabil386) [reply styles/stepper.css]: it has its own css now

File: styles/stepper.css (hunk 2 above)
Comment (nabil386) [reply styles/stepper.css]: .RCP {}
.RCP__track {}
.RCP__progress {}
.RCP__pointer {}
are styles of react-customizable-progressbar component and .RCP alone was a leftover and it's not needed. removed it

File: styles/stepper.css (hunk 3 above)
Comment (nabil386) [reply styles/stepper.css]: done

File: styles/stepper.css [hunk 4]
```diff
@@ -104,3 +104,4 @@
         padding-left: 1rem;
     }
 }
+
```
Comment (arsanig) [inline styles/stepper.css:107]: How come this is a part of the PR? Don't think this needs to be here if there isn't any changes to this file

Comment (nabil386) [reply styles/stepper.css]: file got reverted

File: styles/stepper.css [hunk 5]
```diff
@@ -9,7 +9,7 @@
     margin-bottom: 16px;
     margin-left: 10px;
 }
-  
+
```
Comment (arsanig) [inline styles/stepper.css:12]: Still not sure if this is needed as part of the PR if there are no changes to this file

File: components/atoms/CircleProgressBar.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+export class CircleProgressBar extends Component {
+  render() {
+    return (
+      <div className="absolute inset-x-6 top-9 h-16">
+        <ProgressBar
+          progress={this.props.progress}
+          steps={this.props.steps}
+          radius={28}
+          strokeColor={'#259F46'}
```
Comment (arsanig) [inline components/atoms/CircleProgressBar.js:14]: Can you add this colour to the `tailwind.config.js`?

**Review by arsanig — CHANGES_REQUESTED** [review]
Amazingly clean compared to the custom .css sheet we had before 👏

Other fix - the `package-lock` is built with an old npm, can you update your npm locally and then push those changes?

1. `npm i -g npm@8.4.0`
2. `npm audit fix`
3. `npm i`

**Comment by nabil386** [comment]
what do we do with that color if we added to tailwind ?

**Review by arsanig — APPROVED** [review]

**Pull request merged**